qdrant-landing/content/documentation/concepts/hybrid-queries.md

```diff
@@ -1065,3 +1065,106 @@ In this example, we first fetch 10 points with the color `"red"` and then 10 poi
 Then, we order the results by the price field.
 
 This is how we can guarantee even sampling of both colors in the results and also get the cheapest ones first.
+
+## Grouping
+
+*Available as of v1.11.0*
+
+It is possible to group results by a certain field. This is useful when you have multiple points for the same item, and you want to avoid redundancy of the same item in the results.
+
+REST API ([Schema](https://api.qdrant.tech/master/api-reference/query/query_points_groups)):
```

Comment:
Not up yet.

Reply:
https://github.com/qdrant/qdrant/blob/master/docs/redoc/master/openapi.json#L5460